datasets/meldingen/dataset.json

```diff
@@ -41,7 +41,7 @@
   },
   "tables": [
     {
-      "id": "meldingenbuurt",
+      "id": "meldingenBuurt",
```

Comment:
!

Dit roept om meer validatie.

Reply:
De tabel zelf had al als id `meldingenBuurt`, maar `meldingen/dataset.json` had een mismatch die is gefixt. De tabellen hadden al `meldingenBuurt`, en nu nog steeds. Ik heb iig de output van ` schema create sql datasets/meldingen/dataset.json  --db-url="postgres://dataservices:insecure@0.0.0.0:5415/dataservices"` voor - en na deze update gecheckt en dat is identiek.

Wat voor verdere validatie had je in gedachten?